quiz/classes/RadioQuestion.class.ts

```diff
@@ -0,0 +1,22 @@
+import { Question } from './Question.class';
+import { Printer } from '../interfaces/Printer.interface';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class RadioQuestion extends Question {
+  options: number;
+
+  constructor(private printer: Printer, options, question, answer) {
+    super(question, answer);
+    this.options = options;
+  }
+
+  optionsCount() {
```

Comment:
Would be great if it would more like in real life 🙂 
For example, options would be `options: Array<{ label: string, value: any}>`

`optionsCount` would return `this.options.length`

`print` would return also a list of options, maybe with help of `JSON.stringify`